app/javascript/components/map/MapPopup.vue

```diff
@@ -35,18 +36,7 @@ export default {
       id: this.content.id,
       chartData: Object,
       url: '',
-      colors: [
-        "#97001F",
-        "#6054BA",
-        "#00483A",
-        "#E7C802",
-        "#4bc0c0",
-        "#43B2ED",
-        "#6380ff",
-        "#ff6384",
-        "#ffa040",
-        "#003e78",
-      ],
+      colors: [],
```

Comment:
The constant is `CHART_COLOURS` (with a 'u') but the state key here is `colors` (without a 'u')